lib/blockchain/install.js

```diff
@@ -97,13 +97,17 @@ function plugin (crypto) {
 }
 
 function run () {
+  const ETH_CRYPTO_CODE ='ETH'
   const choices = _.map(c => {
     const checked = isInstalledSoftware(c) && isInstalledVolume(c)
     return {
       name: c.display,
       value: c.code,
       checked,
-      disabled: checked && 'Installed'
+      disabled: z => {
+        if (checked || c.cryptoCode.toString() === ETH_CRYPTO_CODE)
+          return 'Installed, use Infura option'
+      }
```

Comment:
I think this will show the wrong message if the coin is not ETH and is already checked (installed). Could do this (up above):

```javascript
const disabled = c.cryptoCode.toString() === 'ETH'
  ? 'Installed, use Infura option' 
  : checked && 'Installed'
```

Reply:
good suggestion.